corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -263,7 +263,7 @@
                 <div class="accordion-body collapse 
                     {% if cases|length == 1 %}in{% endif %}" 
                     id="form-case-acc-{{ forloop.counter }}">
-                    <div class="accordion-inner">
+                    <div class="accordion-inner" style="display: inline-block">
```

Comment:
Woah, Thanks for explaining the float magic!

I think the `property_table` is used in few [other places](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/templatetags/proptable_tags.py#L211), and changing style rule for it could have unintended consequences.